src/store/store-flag.d.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */
+// THIS FEATURE-FLAG FILE IS AUTOGENERATED,
+//  REMOVAL OR CHANGES WILL CAUSE RELATED TYPES TO STOP WORKING
+import "quasar/dist/types/feature-flag";
+
+declare module "quasar/dist/types/feature-flag" {
+  interface QuasarFeatureFlags {
+    store: true;
+  }
+}
```

Comment:
Indents.

Reply:
I changed it in other places but not there since this file explicitly says not to edit it. It probably wouldn't hurt anything to change spacing but best to not poke it, heh.